py/CHANGELOG.md

```diff
@@ -3,11 +3,13 @@
 ## Unreleased
 
 - Add `thread.state` field to protocol. ([#1896](https://github.com/getsentry/relay/pull/1896))
+- Smart trim loggers for Java platforms. ([#1941](https://github.com/getsentry/relay/pull/1941))
+- Perform PII scrubbing on meta's original_value field. ([#1892](https://github.com/getsentry/relay/pull/1892))
 - PII scrub `span.data` by default. ([#1953](https://github.com/getsentry/relay/pull/1953))
 - Scrub sensitive cookies. ([#1951](https://github.com/getsentry/relay/pull/1951))
-- Apply transaction clustering rules before UUID scrubbing rules. ([#1964](https://github.com/getsentry/relay/pull/1964))
```

Comment:
The Python library does not apply clustering rules, so this change is not relevant to the library: https://github.com/getsentry/relay/blob/f120fbc101d190e5627cedc2d2a43d8f3c88cce9/relay-cabi/src/processing.rs#L126.